openmetadata-ui/src/main/resources/ui/playwright/utils/explore.ts

```diff
@@ -116,3 +123,28 @@ export const selectDataAssetFilter = async (
   await page.getByTestId(`${filterValue}-checkbox`).check();
   await page.getByTestId('update-btn').click();
 };
+
+export const validateBucketsForIndex = async (page: Page, index: string) => {
+  const { apiContext } = await getApiContext(page);
+
+  const response = await apiContext
+    .get(
+      `/api/v1/search/query?q=&index=${index}&from=0&size=10&deleted=false&query_filter=%7B%22query%22:%7B%22bool%22:%7B%7D%7D%7D&sort_field=totalVotes&sort_order=desc`
+    )
+    .then((res) => res.json());
+
+  const buckets = response.aggregations?.['sterms#entityType']?.buckets ?? [];
+
+  // Verify all expected buckets exist and have doc_count > 0
+  const invalidBuckets = EXPECTED_BUCKETS.filter((expectedKey) => {
+    const bucket = buckets.find((b: Bucket) => b.key === expectedKey);
+
+    return !bucket || bucket.doc_count <= 0;
+  });
```

Comment:
instead of returning, we should an expect that this bucket exists.
It gives us clear log when we are checking in CI

Reply:
Added expect that the bucket should exists and its doc_count to be greater than 0